index.html

```diff
@@ -0,0 +1 @@
+Hello world.
```

Comment:
Chris I'd really like to see an exclamation point instead! Please update the code before resubmitting. Cheers mate!

Reply:
I have completed your request sir

Reply:
you sir, are, the best! SHIP IT!!!